services/symbiosis/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11
+WORKDIR /atelier
+
+COPY services/symbiosis/requirements.txt /atelier/requirements.txt
+RUN pip install --no-cache-dir --upgrade -r /atelier/requirements.txt
+
+COPY packages/symbiosis /atelier/symbiosis
+COPY packages/communal /atelier/communal
+
+CMD ["fastapi", "run", "symbiosis/entrypoint.py", "--port", "33333", "--workers", "1"]
```

Comment:
@josiahdc Why not start out with a more robust process manager that can likely work in production? I think `gunicorn -k uvicorn.workers.UvicornWorker` tends to be strictly preferred, though obviously we could spin out a separate Jira issue for tuning the number of threads and workers based on, e.g., host memory. (I presume this doesn't matter a ton during testing without concurrency, connection timeouts, etc., but in production I presume we'll need to be a bit more robust, even with just a few users.)

Reply:
Great catch @chrislrobert - I just pushed a fix!